pulsar-client-tools/src/main/java/com/yahoo/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -48,9 +48,23 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Update dynamic-serviceConfiguration of broker")
```

Comment:
Also, it would be useful to have command to print all the config that are stored in ZK. To know what is being overridden.